simtools/db_handler.py

```diff
@@ -5,9 +5,16 @@
 import yaml
 from pathlib import Path
 
+import simtools.config as cfg
+from simtools.util import names
+from simtools.util.model import validateModelParameter, getTelescopeSize
+
 __all__ = ['getArrayDB']
 
 
+logger = logging.getLogger(__name__)
```

Comment:
I don't see how we could do that in this case. It's ok to pass the name of the logger to a class. But what about modules which are just a collection of functions? We can't give the logger name to each function.

Reply:
Why not? That's how I do it. It can be an optional argument if you want, where if it is not given, then you use __name__.

Reply:
I'm very much against this idea. It's going to be a lof of pollution to add this to all the functions.
We should check how ctapipe does it. I can't believe there is not better way.

Reply:
OK, before I implement the logger everywhere, I will look around on how to do this. Though I must say I already did when I wrote the first example and that is what I found. With classes it's easy, with a collection of functions maybe we can find a different solution. For now we can keep like this, but there's no way that in the long run this would work. You have to allow the user to set the logging level, it's basic functionality.